shims/spark32/src/main/scala/org/apache/spark/sql/execution/datasources/orc/OrcFileFormat.scala

```diff
@@ -83,7 +83,7 @@ class OrcFileFormat extends FileFormat with DataSourceRegister with Serializable
       options: Map[String, String],
       files: Seq[FileStatus]): Option[StructType] = {
     // Why if (false)? Such code requires comments when being written.
-    if ("true".equals(sparkSession.sparkContext.getLocalProperty("isNativeAppliable")) && false) {
+    if ("true" == sparkSession.sparkContext.getLocalProperty("isNativeApplicable") && false) {
```

Comment:
Would it be more straight forward to use `sparkSession.sparkContext.getLocalProperty("isNativeApplicable").toBoolean`?

Reply:
@zhztheplayer, the property can be null and needs extra check if we use that. Maybe, just keep the current change.

> I am still a little bit concerned about the use of these local properties. Perhaps we can manage to remove them later some time.

Yes, it's not a good approach to use local property.

Reply:
Can we use true == xxx? why do we use string "true", is it from gluten config?

Reply:
@FelixYBW, it's the raw value we obtained for this property.
The property value is kept by spark context and it is set by Gluten in `GlutenWriterColumnarRules`.